internal/strategies/gradualrolloutuserid.go

```diff
@@ -0,0 +1,44 @@
+package strategies
```

Comment:
@ivarconr This was more personal preference that any convention. I don't have strong feelings about it so we can change it to be more inline with the other repos. Files containing test cases that should not be compiled into the binary have an `_test` suffix so we can probably use underscores between words in filenames? That pattern seems to be used in other Go projects at least and then we avoid mixing hyphens and underscores. Agreed?